druid/README.md

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
Does this work? If it's a one-line path reference, it works like a symlink?

Reply:
this *is* a symlink, it's just showing up funny in the diff? Unless I really messed something up

Reply:
Sounds like it will be exciting then, to see if everything works when the crate is published :)